### PR TITLE
✨ GitHub issue embed

### PR DESCRIPTION
**Pull request by Sigmanificient**
image

### CODE REVIEW

File: pincer_bot/cogs/info.py [hunk 1]
```diff
@@ -92,5 +94,35 @@ async def pypi_dl_command(self):
             value=f'```py\n{rotated}\n```'
         )
 
+    @Client.event
+    async def on_message(self, message: UserMessage):
+        if "##" not in message.content:
```
Comment (Lunarmagpie) [inline pincer_bot/cogs/info.py:99]: should we allow '#' :thinking:

**Review by Lunarmagpie — APPROVED** [review]

File: pincer_bot/cogs/info.py (hunk 1 above)
Comment (trag1c) [reply pincer_bot/cogs/info.py:99]: Honestly I'd allow it + make `##` not embed it, kinda like an escape sequence :thinking:

Comment (Lunarmagpie) [reply pincer_bot/cogs/info.py:99]: Or even just use \# for that. Discord doesn't render the slash.

**Review by Endercheif — APPROVED** [review]

**Review by trag1c — APPROVED** [review]

**Comment by Lunarmagpie** [comment]
I'm gonna merge this and make the changes to `#` in a new PR

**Pull request merged**